### ISSUE TITLE
Keyboard suggestions always shown on iPhone

### ISSUE THREAD

**Issue by hexise** [ajaxorg/ace#4617]
I am using iPhone Safari to open the ace editor at: https://ace.c9.io/build/kitchen-sink.html

I found that even the text area "ace_text-input" already have been marked as autocorrect="off" autocapitalize="off" spellcheck="false" in lib/ace/keyboard/textinput.js, the soft keyboard still have the input suggestions, as shown in the screenshot, I mark the part in red.

I found that there is an old issue for this behavior: https://github.com/ajaxorg/ace/issues/2262

I want to make the suggestion bar hidden, is this the correct way? Thanks.

![ios_input_suggestion](https://user-images.githubusercontent.com/6280560/150036336-adc6e8c5-4c67-4817-9dff-a693148dcbac.png)

**Comment by nightwing** [comment]
The fix suggested in the linked issue is already applied to ace, and On iphone 12 emulator i don't get the suggestion bar, which version are you using?

Unfortunately i am not aware of any other thing that can be done from js to hide the suggestion bar, do you get the suggestion bar displayed with regular textarea as well?

**Comment by rhmbeta** [comment]
I am using iPhone XS with IOS 15.2.

I also tested on emulator running IOS 15.2 and the suggestion shows, too.

I also tried the standard text area with autocorrect="off" autocapitalize="off" spellcheck="false", the suggestion still shows. So maybe there are other places to control it?

![Screen Shot 2022-01-20 at 13 32 01](https://user-images.githubusercontent.com/97474205/150425567-2f075152-4820-4fe6-86c0-e642327ae282.png)

![Screen Shot 2022-01-20 at 13 27 35](https://user-images.githubusercontent.com/97474205/150425049-34ebfee9-5111-4f4b-bbe8-6533e25e66d1.png)
